qutip/core/coefficient.py

```diff
@@ -0,0 +1,621 @@
+import numpy as np
+import scipy
+import os
+import sys
+import re
+import dis
+import hashlib
+import glob
+import importlib
+import shutil
+from .. import settings as qset
+from .data import Data
+from .interpolate import Cubic_Spline
+from .cy.coefficient import (InterpolateCoefficient, InterCoefficient,
+                             StepCoefficient, FunctionCoefficient,
+                             SumCoefficient, MulCoefficient,
+                             ConjCoefficient, NormCoefficient,
+                             ShiftCoefficient)
+from setuptools import setup, Extension
+from Cython.Build import cythonize
```

Comment:
I think this will error `import qutip` if Cython is not installed (`core.qobjevo` imports this file, so it'll happen).  Probably just needs this import moving into the function where it's used, and have the import `try; except ImportError` guarded.

Reply:
Forgot that, thank you for reminding me.